activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-rest/src/main/java/org/activiti/cloud/services/modeling/rest/controller/ModelingRestExceptionHandler.java

```diff
@@ -87,6 +88,13 @@ public void handleModelNameConflictException(ModelNameConflictException ex,
         response.sendError(CONFLICT.value(), ex.getMessage());
     }
 
+    @ExceptionHandler(ModelNameInvalidException.class)
+    public void handleModelNameInvalidException(ModelNameInvalidException ex,
+                                                 HttpServletResponse response) throws IOException {
+        logger.error(ex.getMessage(), ex);
+        response.sendError(CONFLICT.value(), ex.getMessage());
+    }
```

Comment:
You are doing the same as the previous exception handler so I think it is better to merge the three we have for sending the conflict like this
```java
@ExceptionHandler({
    ModelNameConflictException.class,
    ModelNameInvalidException.class,
    ModelScopeIntegrityException.class
})
```